source/about/security/mobile-security.rst

```diff
@@ -0,0 +1,48 @@
+Mobile Security
+================
+
+Mattermost mobile is built with a robust security framework to protect user data and prevent unauthorized access. The Mattermost Mobile app's comprehensive security framework ensures that user data remains protected against unauthorized access.
+
+Continuous compliance through :ref:`secure SDLC practices and proactive vulnerability management <deploy/mobile/mobile-security-features:security compliance>` further reinforces the platform’s resilience.
+
+As the mobile app evolves with future enhancements like file sharing restrictions, Mattermost is committed to delivering a secure, reliable mobile experience.
```

Comment:
Do we want to share the file sharing restrictions possible enhancements in our docs?